Emby.Server.Implementations/LiveTv/Listings/XmlTvListingsProvider.cs

```diff
@@ -17,7 +17,7 @@
 using MediaBrowser.Model.LiveTv;
 using Microsoft.Extensions.Logging;
 
-namespace Jellyfin.Server.Implementations.LiveTv.Listings
+namespace Emby.Server.Implementations.LiveTv.Listings
```

Comment:
I'm not sure about this one.

Reply:
I personally would rather see the `Emby.Server.Implementations` renamed to `Jellyfin.Server.Implementations` if that is not too much difficult...

Reply:
We have to be careful if we do that, resources use the name of the csproj